reactive/kubernetes_master.py

```diff
@@ -3658,11 +3696,73 @@ def configure_kubelet():
     kubernetes_common.configure_kubelet(
         dns_domain, dns_ip, registry, taints=taints, has_xcp=has_xcp
     )
+    set_state("kubernetes-master.config.created")
     service_restart("snap.kubelet.daemon")
-
+    set_state("kubernetes-master.label-config-required")
```

Comment:
Do we need to reconfigure labels every time Kubelet restarts? Once the node has been registered, Kubelet shouldn't be changing its own labels.

I think really, we just need to set the labels one time after the Node has been created. Then update them any time the charm is upgraded (in case the charm's label handling code changed) or when the labels config changed.

Reply:
maybe?  again... was just following what k8s-worker was doing